tests/UI/campaigns/functional/BO/03_catalog/07_discounts/01_cartRules/01_CRUDCartRule.js

```diff
@@ -265,7 +265,8 @@ describe('CRUD cart rule', async () => {
     it('should verify the total after discount', async function () {
       await testContext.addContextItem(this, 'testIdentifier', 'verifyTotalAfterDiscount_2', baseContext);
 
-      const discountedPrice = Products.demo_1.finalPrice - (Products.demo_1.finalPrice * editCartRuleData.value / 100);
+      const discountedPrice = Products.demo_1.finalPrice
+        - (Products.demo_1.finalPrice * editCartRuleData.discountPercent / 100);
```

Comment:
Javascript is notorious for having a weird way to calculate with float, so let's pray it works exactly like in the PHP code of the core :pray: